src/models/TuNewsModel.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import type Moment from 'moment'
+
+class TuNewsModel {
+  _id: number
+  _title: string
+  _tags: string[]
+  _date: Moment
+  _content: string
+  _enewsno: string
+
+  constructor (params: {| id: number, title: string,
+    date: Moment, tags: string[], content: string, enewsno: string
+    |}) {
+    const {id, date, title, tags, content, enewsno} = params
+    this._id = id
+    this._title = title
+    this._tags = tags
+    this._date = date
+    this._content = content
+    this._enewsno = enewsno
+  }
+
+  get id (): number {
+    return this._id
+  }
+
+  get title (): string {
+    return this._title
+  }
+
+  get date (): Moment {
+    return this._date
+  }
+
+  get tags (): string[] {
```

Comment:
Please use `Array<string>` for consistency. Same for other occurences.

Reply:
Fixed